charts/dev/theme-park/questions.yaml

```
@@ -0,0 +1,306 @@
+# Include{groups}
+portals:
+  open:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+questions:
+  - variable: portal
+    group: "Container Image"
+    label: "Configure Portal Button"
+    schema:
+      type: dict
+      hidden: true
+      attrs:
+        - variable: enabled
+          label: "Enable"
+          description: "enable the portal button"
+          schema:
+            hidden: true
+            editable: false
+            type: boolean
+            default: true
```

```suggestion
portals: {}

questions:
```

I think this does not have a webui right?
IF that's true, lets not display an "open" button

---

If it actually has a webUI, ignore that

---

It has a WebUI that is the same as https://theme-park.dev